Mission build tool and Travis-CI integration

veteran29 opened the pull request:
As the amount of combinations of available _maps_ and _presets_ is quite big I think Liberation could use an build tool.

In this pull request I have implemented _gulpjs_ mission builder. Main task of this tool is creation of PBOs and release ZIP files in `_tools/build/` folder.

Based on entries in `_tools/_presets.json` it builds multiple _missions_ and configures them accordingly. 

Currently it is configured for 0.963, however I had in mind that 0.97 is coming in future so I tried to make it easily adjustable.

In conjunction with Travis-CI it can "automagicaly" build mission release files and attach them to tags.
I have set up `.travis.yml` to only run when new tag is pushed to repository as there are no tests or linting.   
<sup>( for now :D )</sup>

Example [tag](https://github.com/veteran29/kp_liberation/releases/tag/v0.963-additions-travis) with files attached by travis and [build log](https://travis-ci.org/veteran29/kp_liberation/builds/340742325)

[Readme](https://github.com/veteran29/kp_liberation/tree/feature/gulp-mission-tool/_tools) can be found in root of `_tools` folder.
  
  
I have set up `_presets.json` to build missions with same configuration as files attached to v0.963 tag, with two differences:
  * Chernarus uses GREF resistance instead of Vanilla
  * Added `kp_liberation_afrf.Takistan` version with AFRF preset in place of USAF. This shows how this can be used to build multiple variants of mission on same map easily.
  

Feel free to message me on Discord if you need some help with Travis or want to discuss.

Wyqer commented on .gitignore:
```diff
@@ -45,3 +45,10 @@ $RECYCLE.BIN/
 Network Trash Folder
 Temporary Items
 .apdisk
+
```
Are changes to the .gitignore really necessary?

veteran29 replied:
I think so. If node_modules will be not ignored every git will see every dependency as new file.

veteran29 replied:
Just thought about this, these rules could be moved to separate .gitignore in _tools subdirectory.

veteran29 commented:
I have rebased my branch on _0.963-additions_ branch to resolve conflicts.
Also I have moved ignore rules for build tool to separate .gitignore in "_tools" directory.
